docs/source/index.rst

```diff
@@ -96,3 +96,9 @@ Indices and tables
 .. |license| image:: https://img.shields.io/github/license/indico/indico.png
                    :alt: License
                    :target: https://github.com/indico/indico/blob/master/COPYING
+
+Contact
+++++++++++++++++++
+
+* IRC Channel: Official `#indico <https://webchat.freenode.net/?channels=indico>`_ channel at Freenode (irc.freenode.net). Also accessible through `Matrix <https://riot.im/app/#/room/#indico:matrix.org>`_.
+* Discussion forum: `Indico Community <https://talk.getindico.io>`_ Discourse instance
```

Comment:
is the fact that it's a Discourse instance really relevant?

Reply:
Just copied it from https://getindico.io/contact/, but I agree it's no really relevant.